content/3_Silver/Prefix_Sums.mdx

```diff
@@ -401,14 +401,60 @@ vl psum(const vl& a) {
 }
 
 int main() {
-	for (ll i: psum({1,2,3,4,5}))
-		cout << i << " ";
-	// 0 1 3 6 10 15
+	int N, Q;
+	cin >> N >> Q;
+	vl a(N);
+	for (ll& x: a)
+		cin >> x;
+	vl p = psum(a);
+	for (int i = 0; i < Q; ++i) {
+		int l, r;
+		cin >> l >> r;
+		cout << p[r] - p[l] << "\n";
+	}
 }
 ```
 
 </CPPSection>
 
+<JavaSection>
+
+```java
+import java.util.*;
+
+public class Main{
+    static int N, Q;
+    public static void main(String[] args){
+        Scanner in = new Scanner(System.in);
```

Comment:
Why use Scanner (not a big issue but i don't like spreading around the use of Scanner)